Gopkg.toml

```diff
@@ -147,7 +147,8 @@
 
 [[constraint]]
   name = "gopkg.in/juju/charm.v6"
-  revision = "adaf8c5c203723a78e5d6e54492505c574bdf28c"
+  revision = "a595818559134f9e9981a08a32cd9de76cb478f0"
+  source = "github.com/nammn/charm"
```

Comment:
Once this pr has been approved ill merge the charm package first and then update this pr